README.md

```diff
@@ -61,8 +61,8 @@ to `./runner.yml` and overriding relevant properties.
 
 To set up the project run:
 
-```
-$ ./vendor/bin/run drupal:site-setup
+```bash
+./vendor/bin/run drupal:site-setup
```

Comment:
This should be removed like in other repositories we are now updating.

Reply:
Fixed.